cmd/ebrelayer/txs/registry.go

```diff
@@ -0,0 +1,88 @@
+package txs
+
+import (
+	"context"
+	"log"
+
+	"github.com/ethereum/go-ethereum/accounts/abi/bind"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/ethclient"
+
+	bridgeRegistry "github.com/cosmos/peggy/cmd/ebrelayer/generated/bridgeregistry"
+)
+
+// ContractRegistry :
+type ContractRegistry int
```

Comment:
you can use `byte` here instead
```suggestion
type ContractRegistry byte
```

Reply:
https://github.com/cosmos/peggy/pull/77/commits/be4c43930db5f73c577d2a55782bf464405dd549